R/load.R

```diff
@@ -21,4 +21,15 @@ stone_load <- function(transformed_data, con,
   load_touchstone(transformed_data, con)
   load_scenario_description(transformed_data, con,
                             allow_overwrite_scenario_description)
+  load_touchstone_demographic_dataset(transformed_data, con)
+
+  if (!is.null(con)) {
```

Comment:
Consider moving this into its own function: `check_faulty_serials`

Reply:
Have merged it into one function in utils.